torchao/_models/_eval.py

```diff
@@ -272,91 +163,47 @@ def record_inputs(
         )
         return self
 
-    def get_inputs(self):
-        return self.inputs
+    def get_inputs(self):  # for BC
+        return self.get_recorded_inputs()
+
+    def get_recorded_inputs(self):
+        return self.base_input_recorder.get_recorded_inputs()
+
+    def get_recorded_args_and_kwargs(self):
+        return self.base_input_recorder.get_recorded_args_and_kwargs()
```

Comment:
nit: get_recorded_args(self) might be more consistent with get_recorded_args_and_kwargs?

does recorded inputs support recording multiple args?

Reply:
i wanted to maintain the old API which is used in a few places just to get data.